plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/type/TestTypeInfoUtils.java

```diff
@@ -0,0 +1,186 @@
+/*
```

Comment:
Could you add new tests for special column names, unsupported column/field types to BaseBigQueryConnectorTest? You can refer to `testStreamCommentTableSpecialCharacter` method.

Reply:
I found an existing `testSkipUnsupportedType` and added a query to `information_schema.columns` without any predicates, to make sure the new override method is used. The new `TestBigQueryType.testConvertToTrinoType()` covers all supported and unsupported data types.